charts/e2g/templates/bundle.yaml

```
@@ -57,21 +57,49 @@ resources:
         name: e2g-config
         labels: {{- include "lib.utils.common.labels" $| nindent 10 }}
       data:
-    {{- if $.Values.e2g.story }}
-      {{- range $i, $e := $.Values.e2g.story }}
-        {{ $i }}.story: |-
-          {{ $e | nindent 10 }}
-      {{- end -}}
-    {{- end -}}
-    {{- if $.Values.e2g.config }}
         e2guardian.conf: |-
-          {{ $.Values.e2g.config | nindent 10 }}
-    {{- end -}}
-    {{- if $.Values.e2g.lists }}
-      {{- range $i, $e := $.Values.e2g.lists }}
+    {{ $.Values.e2g.config | nindent 10 }}
+  {{- end -}}
+  {{- if $.Values.e2g.lists }}
+  - type: "raw"
+    manifest: |
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: e2g-lists
+        labels: {{- include "lib.utils.common.labels" $| nindent 10 }}
+      data:
+    {{- range $i, $e := $.Values.e2g.lists }}
         {{ $i }}.list: |-
           {{ $e | nindent 10 }}
-      {{- end -}}
+    {{- end -}}
+  {{- end -}}
+  {{- if $.Values.e2g.story }}
+  - type: "raw"
+    manifest: |
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: e2g-story
+        labels: {{- include "lib.utils.common.labels" $| nindent 10 }}
+      data:
+    {{- range $i, $e := $.Values.e2g.story }}
+        {{ $i }}.story: |-
+          {{ $e | nindent 10 }}
+    {{- end -}}
+  {{- end -}}
+  {{- if  $.Values.e2g.filtergroups }}
+  - type: "raw"
+    manifest: |
+      apiVersion: v1
+      kind: ConfigMap
+      metadata:
+        name: e2g-filtergroups
+        labels: {{- include "lib.utils.common.labels" $| nindent 10 }}
+      data:
+    {{- range $i, $e := $.Values.e2g.filtergroups }}
+        e2guardianf{{ .id }}.conf: |-
+          {{ .config | nindent 10 }}
     {{- end -}}
```

Why do you assign vars `$i` and `$e` but just access `.id` and `.config`? You don't need the variables therefor you don't have to assign any variables.

```suggestion
    {{- range $.Values.e2g.filtergroups }}
        e2guardianf{{ .id }}.conf: |-
          {{ .config | nindent 10 }}
    {{- end -}}
```

---

Or did you mean to use the variables? Because in the other scenarios you used it correctly.